docs/guides/v8-upgrade-guide.md

```diff
@@ -13,17 +13,46 @@ The main feature for this release is the switch of the theming engine, we replac
 
 We recommend upgrading your application for each major version gradually, e.g. if you plan to update from 5.x to 8.x migrate first to 6.x, then 7.x and finally to version 8.
 
-## Why we switched from emotion to themeable
+```javascript
+---
+embed: true
+---
+<ToggleBlockquote
+  summary="Why we switched from emotion to themeable"
+>
+  <ToggleBlockquote.Paragraph>
+    We feel that it is very important that InstUI is easy to use and learn. To accomplish this, we try to use well-known open source solutions whenever possible.
+  </ToggleBlockquote.Paragraph>
+  <ToggleBlockquote.Paragraph>
+    The previous, custom made theming solution did not fit into this vision (for its defense it was added when there weren't widely used alternatives)
+    so we decided to switch to a broadly adopted one. The two main candidates were <Link href="https://emotion.sh">emotion.sh</Link> and <Link href="https://styled-components.com">styled components</Link>, we decided to use emotion.
+  </ToggleBlockquote.Paragraph>
+  <ToggleBlockquote.Paragraph>
+    If you are interested in our detailed reasoning you can read it <Link href="https://gist.github.com/serikjensen/4ba00b653efac1dbf80543c529adabbc">here</Link>.
+  </ToggleBlockquote.Paragraph>
+</ToggleBlockquote>
+```
+
+## Before you upgrade to v8.0
+
+There are a few steps you need to do before you start the migration process. **First, upgrade to the latest v7.x release to have access to all the new components and features.**
```

Comment:
I don't what you mean by this part: "to have access to all the new components and features". I'd just write:

"**First, upgrade to the latest v7.x release to be able to follow this guide.**"

Reply:
I kinda "stole it" from the v7 migration guide, and also, we have a new component (TreeBrowser.Node), but probably your version works well too. Changed it.